react-calculator/src/hooks/useCalculate.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import { OperationType } from "../components/Opertaions";
+
+const useCalculate = () => {
+  const [firstNumber, setFirstNumber] = useState("");
+  const [secondNumber, setSecondNumber] = useState("");
+  const [resultNumber, setResultNumber] = useState(0);
+  const [operation, setOperation] = useState<OperationType>();
+  const [done, setDone] = useState(false);
+
+  const state = {
+    firstNumber,
+    secondNumber,
+    resultNumber,
+    operation,
+    done,
+  };
+
+  const firstNumberHandler = (num: number) => {
+    if (firstNumber.length >= 3) return;
+    const number = firstNumber + num;
+    setFirstNumber(number);
+  };
+  const secondNumberHandler = (num: number) => {
+    if (secondNumber.length >= 3) return;
+    const number = secondNumber + num;
+    setSecondNumber(number);
+  };
+
+  const numHandler = (num: number) => {
+    if (!operation) {
+      firstNumberHandler(num);
+      return;
+    }
+    secondNumberHandler(num);
+  };
+
+  const getResultValue = () => {
+    if (!isFinite(resultNumber)) return "오류";
+    if (done) return Math.floor(resultNumber).toString();
+    if (secondNumber) {
+      return secondNumber;
+    } else if (firstNumber) {
+      return firstNumber;
+    }
```

Comment:
해당 부분도 else if를 제거해주면 가독성이 높아질 것 같습니다!

```suggestion
    if (secondNumber) return secondNumber;
    if (firstNumber) return firstNumber;
```